mongodb/intro/date-aggregation/aggregation-with-date-expression-operators-ii.md

```diff
@@ -0,0 +1,172 @@
+---
+author: Stefan-Stojanovic
+
+levels:
```

Comment:
I don't think these are needed anymore, we should remove them

Reply:
@Stefan-Stojanovic can you remove the levels?